<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="greeting">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hello,</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>[USERNAME]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cashierButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Employees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>420</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Inventory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>500</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Promotions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Logs</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>621</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="salesOverview">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Sales Overview</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalLabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
    <widget class="QLabel" name="todayLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Today</string>
     </property>
    </widget>
    <widget class="QLabel" name="todayTotal">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>[TODAY TOTAL]</string>
     </property>
    </widget>
    <widget class="QLabel" name="weekLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>This Week</string>
     </property>
    </widget>
    <widget class="QLabel" name="weeklyTotal">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>[WEEKLY TOTAL]</string>
     </property>
    </widget>
    <widget class="QLabel" name="monthLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>This Month</string>
     </property>
    </widget>
    <widget class="QLabel" name="monthlyTotal">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>130</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>[MONTHLY TOTAL]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Top 5 sellers</string>
     </property>
    </widget>
    <widget class="QLabel" name="topOne">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[FIRST]</string>
     </property>
    </widget>
    <widget class="QLabel" name="topTwo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[SECOND]</string>
     </property>
    </widget>
    <widget class="QLabel" name="topThree">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[THIRD]</string>
     </property>
    </widget>
    <widget class="QLabel" name="topFour">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[FOURTH]</string>
     </property>
    </widget>
    <widget class="QLabel" name="topFive">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[FIFTH]</string>
     </property>
    </widget>
    <widget class="QLabel" name="countOne">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>220</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[COUNT 1]</string>
     </property>
    </widget>
    <widget class="QLabel" name="countTwo">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>250</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[COUNT 2]</string>
     </property>
    </widget>
    <widget class="QLabel" name="countThree">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>280</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[COUNT 3]</string>
     </property>
    </widget>
    <widget class="QLabel" name="countFour">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>320</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[COUNT 4]</string>
     </property>
    </widget>
    <widget class="QLabel" name="countFive">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>370</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[COUNT 5]</string>
     </property>
    </widget>
    <widget class="QLabel" name="yearLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>This Year</string>
     </property>
    </widget>
    <widget class="QLabel" name="yearlyTotal">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>[YEARLY TOTAL]</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistics</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>55</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>pls group this. wla ni nka frame. igo rani ga fetch ug data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>logout</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
